stacks-signer/Cargo.toml

```diff
@@ -42,7 +42,7 @@ thiserror = "1.0"
 toml = "0.5.6"
 tracing = "0.1.37"
 tracing-subscriber = { version = "0.3.17", features = ["env-filter"] }
-wsts = "4.0.0"
+wsts = { version = "4.0.0", default-features = false }
```

Comment:
Is there a reason why we have such an old version of `wsts` here in the `stacks-signer` `Cargo.toml`?  And why we have different versions here and in the root `Cargo.toml`?

Reply:
Can't say for certain, but best to leave as-is since this branch has been genesis-sync tested as working.